src/core/lib/security/transport/secure_endpoint.cc

```diff
@@ -143,12 +163,51 @@ static void secure_endpoint_unref(secure_endpoint* ep) {
 static void secure_endpoint_ref(secure_endpoint* ep) { gpr_ref(&ep->ref); }
 #endif
 
+static void post_reclaimer(secure_endpoint* ep) {
```

Comment:
suggest `maybe_post_reclaimer` to signal this isn't done unconditionally

Reply:
Sure, done